api/maistra/v1alpha1/session_types_test.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha1_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/maistra/istio-workspace/api/maistra/v1alpha1"
+)
+
+var _ = Describe("API manipulation", func() {
+
+	Context("when flipping readiness", func() {
+
+		var components v1alpha1.StatusComponents
+		compoentOne := "one"
+		compoentTwo := "two"
+
```

Comment:
Nit-picking... but... you have a typo `component` :)

I would normally do suggestion, but it's easier to rename in the IDE :)

Reply:
fixed https://github.com/maistra/istio-workspace/pull/875/commits/ecfa131c39f898fb898c5782eae312a1f080bc75